### PR TITLE
472 generate message when test failed [TO SOLVE]

### PR DESCRIPTION
**Pull request by sergiobeltranguerrero**
Generate a message in the pull request and in discord when a test fails.

### CODE REVIEW

File: tests/unit/SignIn.spec.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('SignIn', () => {
 
   it('should display email input', async () => {
     render(SignInView)
-    expect(screen.getByLabelText(/E-mail/i, { selector: 'input' })).toBeInTheDocument()
+    expect(screen.getByLabelText(/E-mailxxxxx/i, { selector: 'input' })).toBeInTheDocument()
```
[the remark this answers is not in the document]
Comment (sergiobeltranguerrero) [reply tests/unit/SignIn.spec.js:13]: Yesss!

**Comment by sergiobeltranguerrero** [comment]
## Pre-Merge Requirements

### Instructions:

Before merging this pull request, please complete the following steps:

1. **Create a Discord Channel**:
   - Go to your Discord server and create a new channel where you want to receive error notifications.

2. **Obtain the Discord Webhook**:
   - In the Discord channel, go to the channel settings and select "Integrations".
   - Click "Create Webhook" and configure the webhook name and image as desired.
   - Copy the generated webhook URL.

3. **Add the Secret to the GitHub Project**:
   - Go to the repository settings in GitHub.
   - In the "Secrets and variables" section under "Security", select "Actions".
   - Create a new secret named `DISCORD_WEBHOOK` and paste the copied webhook URL.

### Pull Request Details:

This pull request includes an improvement that sends a message to Discord if a GitHub event related to a pull request fails. Here are the technical details of the implementation:

- **Discord Webhook**: Uses the URL stored in the `DISCORD_WEBHOOK` secret.
- **Condition**: Sends the message only if the event is a `pull_request` and if the execution fails.
- **Message Data**: The message includes information about the repository, workflow, actor, link to the pull request, and a link to the failed action details.

```yaml
- name: Send message to Discord
  env:
    DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
  if: failure() && github.event_name == 'pull_request'
  uses: Ilshidur/action-discord@master
  with:
    args: |
      ⚠️ The GitHub event `${{ github.event_name }}` has failed.
      **Repository**: `${{ github.repository }}`
      **Workflow**: `${{ github.workflow }}`
      **Actor**: `${{ github.actor }}`
      **Pull Request**: [Link](${{ github.event.pull_request.html_url }})
      **Action URL**: [View Details](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
      Please review the proposed changes.
```

Please make sure to complete these steps before proceeding with the merge of this pull request.

Thank you.

**Comment by sergiobeltranguerrero** [comment]
image

**Review by jvJUCA — APPROVED** [review]
Pre-accept configuration made, i will accept and test!

**Pull request merged**